loginserver/client.cpp

```diff
@@ -5,8 +5,11 @@
 #include "../common/string_util.h"
 #include "encryption.h"
 #include "account_management.h"
+#include "../common/rulesys.h"
 
 extern LoginServer server;
+int32_t expansion = 0;
+int32_t owned_expansion = 0;
```

Comment:
Both of these can be moved to local variables in `SendExpansionPacketData`

Reply:
Done.